packages/cli/src/package.ts

```diff
@@ -420,6 +422,15 @@ function parseExportsFieldConfig(
           name
         );
       }
+    } else if (key === "unwrappedDefaultExportForImportCondition") {
```

Comment:
I'd love to use this repo-wide, see a similar concern+question [here](https://github.com/preconstruct/preconstruct/pull/542#discussion_r1165762092)

Reply:
Yes to making it project-wide, an empty object should be valid and equivalent to true (i would imagine an empty object already works on a package)